app/js/pages/Escrow/components/CardEscrow.jsx

```diff
@@ -1,13 +1,87 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Card, CardBody, Button } from 'reactstrap';
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faCircleNotch, faCheck} from "@fortawesome/free-solid-svg-icons";
+
+import Reputation from '../../../components/Reputation';
+import RoundedIcon from "../../../ui/RoundedIcon";
+
+import one from "../../../../images/escrow/01.png";
+import two from "../../../../images/escrow/02.png";
+import three from "../../../../images/escrow/03.png";
+import four from "../../../../images/escrow/04.png";
+import five from "../../../../images/escrow/05.png";
+
+const Done = () => (
+  <React.Fragment>
+    <RoundedIcon icon={faCheck} bgColor="green"/>
+    <h2 className="mt-4">Done.</h2>
+    <h2 className="mt-4">Rate your trading experience with this user.</h2>
+    <Reputation reputation={{upCount: 1, downCount: 1}} size="l"/>
+  </React.Fragment>
+);
+
+const Releasing = () => (
+  <React.Fragment>
+    <span className="bg-dark text-white p-3 rounded-circle">
+      <img src={five} alt="five" />
+    </span>
+    <h2 className="mt-4">Waiting for the confirmations from the miners</h2>
+    <FontAwesomeIcon icon={faCircleNotch} size="5x" spin/>
+  </React.Fragment>
+);
+
+const Funded = () => (
+  <React.Fragment>
+    <span className="bg-dark text-white p-3 rounded-circle">
+      <img src={four} alt="four" />
+    </span>
+    <h2 className="mt-4">Funds are in the escrow. Release them when you will get the payment.</h2>
+    <Button color="primary" className="btn-lg mt-3" onClick={() => {}}>Release funds</Button>
+  </React.Fragment>
+);
+
+const Funding = () => (
+  <React.Fragment>
+    <span className="bg-dark text-white p-3 rounded-circle">
+      <img src={three} alt="three" />
+    </span>
+    <h2 className="mt-4">Waiting for the confirmations from the miners</h2>
+    <FontAwesomeIcon icon={faCircleNotch} size="5x" spin/>
+  </React.Fragment>
+);
+
+const PreFund = () => (
+  <React.Fragment>
+    <span className="bg-dark text-white p-3 rounded-circle">
+      <img src={two} alt="two" />
+    </span>
+    <h2 className="mt-4">
+      You are about to send
+      <p className="h2 text-success">2.0 ETH</p>
+      + our fee
+      <p className="h2 text-success">1 SNT</p>
+    </h2>
```

Comment:
Sorry to be that picky, but it should be something like
```
<p className="h2 text-success mt-4" You are about to send</p>
<p className="h2 text-success">2.0 ETH</p>
<p className="h2 text-success">+ our fee</p>
<p className="h2 text-success">1 SNT</p>
```
you don't actually need the `h2`. If you want to wrap it, you can always use a `div`
Putting only one line out of two in `p` is really weird lol

Reply:
It's ok, be picky please!!!
Is it cool now?

Reply:
Yes perfect!